controller.js

```diff
@@ -18,4 +18,10 @@ function getArticleById(req, res, next){
     })
     .catch(next);
 }
-module.exports = {getEndpoints, getAllTopics, getArticleById};
+
+function getAllArticles(req, res, next){
+    getAllArticlesData().then((articleData) =>{
```

Comment:
You are expecting back some articles...what could be a more semantic variable name here